### ISSUE TITLE
String is not parsed and callback not called when <?xml version="1.0"?> is the only string

### ISSUE THREAD

**Issue by coockoo** [Leonidas-from-XIV/node-xml2js#245]
```javascript
var xml2js = require('xml2js');
xml2js.parseString('<?xml version="1.0"?>', function (err, res) {
	console.log(err);
	console.log(res);
})
```
This piece of code does nothing. Callback is not even called

**Comment by tflanagan** [comment]
It looks like we are missing a ```saxParser.onend``` function

**Comment by tflanagan** [comment]
@coockoo what would expect the result to be? Null, undefined, empty string or an empty object?

**Comment by coockoo** [comment]
@tflanagan I think the result should be the same as for
```javascript
xml2js.parseString('', function (err, res) {
    console.log(err);
    console.log(res);
})
```

**Comment by tflanagan** [comment]
@coockoo that returns ```null```

**Comment by coockoo** [comment]
@tflanagan excellent. So if my example returns `null` it would be great.

**Comment by tflanagan** [comment]
Fixed in #246

**Comment by Leonidas-from-XIV** [comment]
Merged fix!
